docs/src/docs/cli/cli-reference.header

```diff
@@ -1,5 +1,8 @@
 ---
 outline: 2
+title: "Kit CLI Reference"
 ---
-# Kit CLI Reference
-
+<script setup>
+  import { version } from '../../../package.json'
+</script>
+# Kit CLI Reference {{ version }}
```

Comment:
We definitely will want to update this :D but later, ill leave this to Annika but i think that having the version in the h1 is not ideal as it changes with each release.

Reply:
Yeah, I wasn't sure where to put it honestly.